style.css

```diff
@@ -0,0 +1,193 @@
+* {
```

Comment:
Noticed how nicely you used some CSS properties in your code. I can visualise what your code is doing but I will try and clone it over the weekend and see the actual view.  Well done!